incountry/validation/validate_encryption_enabled.py

```diff
@@ -1,15 +1,18 @@
-import wrapt
+from functools import wraps
+from inspect import signature
 
-from .utils import function_args_to_kwargs
 from ..exceptions import StorageClientException
 
 
-@wrapt.decorator
-def validate_encryption_enabled(function, instance, args, kwargs):
-    function_args_to_kwargs(function, args, kwargs)
+def validate_encryption_enabled(function):
+    @wraps(function)
+    def inner(*args, **kwargs):
+        class_instance = args[0] if "self" in signature(function).parameters else None
+        if class_instance and not class_instance.encrypt:
+            raise StorageClientException(
+                f"Validation failed during {function.__qualname__}(): "
+                f"This method is only allowed with encryption enabled"
```

Comment:
```suggestion
                f"This method is only allowed with encryption enabled"
```
And this is why we use flake8: this f-string does nothing

Reply:
I'd vote for keeping as is just for consistency. Anyway - will read more on this and try to figure out why the issue wasn't raised by flake8